spec/readthis/cache_spec.rb

```diff
@@ -156,6 +156,13 @@ def self.load(value)
 
       expect(cache.read('short-key')).to eq('other stuff')
     end
+
+    context 'handling passing nil to options' do
```

Comment:
Spec looks right on. Can you drop the `context` and put more detail into the spec description instead?